data/supertypes.yml

```diff
@@ -1,25 +1,50 @@
 # Groups used for navigation.
 navigation_document_supertype:
-  - id: guidance
-    document_types:
-      - answer
-      - contact
-      - detailed_guide
-      - document_collection
-      - form
-      - guidance
-      - guide
-      - licence
-      - local_transaction
-      - manual
-      - map
-      - notice
-      - place
-      - programme
-      - promotional
-      - regulation
-      - simple_smart_answer
-      - smart_answer
-      - statutory_guidance
-      - transaction
-      - travel_advice
+  default: other
+  items:
+    - id: guidance
+      document_types:
+        - answer
+        - contact
+        - detailed_guide
+        - document_collection
+        - form
+        - guidance
+        - guide
+        - licence
+        - local_transaction
+        - manual
+        - map
+        - notice
+        - place
+        - programme
+        - promotional
+        - regulation
+        - simple_smart_answer
+        - smart_answer
+        - statutory_guidance
+        - transaction
+        - travel_advice
+# Used to distinguish page used mainly for navigation from content pages
+user_journey_stage:
```

Comment:
I was planning on naming them `x_document_supertype`, like `whitehall_document_supertype` and `navigation_document_supertype`.

Would it make sense to name this `user_journey_document_supertype`?